docs/docs/02-guide/04-blog/08-play.md

```diff
@@ -3,7 +3,7 @@
 ## Create a blog post by Alice
 
 ```
-blogd tx blog create-post hello world --from alice
+blogd tx blog create-post hello world --from alice --chain-id blog
```

Comment:
Why precising always the chain id? Maybe we should set it in the client.toml first.
`blogd config set client chain-id blog`

Reply:
The queries/txs are not working without the chain-id into the SDK v0.50, and are already set the chain id into the config